config/locales/views/user_mentee_applications/mentee_application_states/en.yml

```diff
@@ -1,5 +1,22 @@
 en:
   application_received:
+    applicant_information: "
+      <h3 class='text-xl mt-4'>What is our Mission?</h3>
+      <p>At AOL, we bridge the gap between learning and doing. We team up junior developers with seasoned mentors to tackle real-world projects, ensuring that every member not only gains invaluable experience but becomes an integral part of a community that champions each other's growth and success in the tech industry.</p>
+      <h3 class='text-xl mt-4'>Who would be a good fit?</h3>
+      <ol class='list-decimal list-inside'>
+        <li><span class='font-bold'>Aspiring Developers</span>: Individuals who are passionate about coding and technology, looking for real-world experience. They could be recent graduates, self-taught programmers, or career switchers aiming to break into the tech industry.</li>
+        <li><span class='font-bold'>Continuous Learners</span>: Those who have a genuine curiosity and drive to continuously upskill, learn from others, and share their knowledge.</li>
+        <li><span class='font-bold'>Team Players</span>: Given the collaborative nature of projects, individuals who communicate well, are open to feedback, and can work effectively in a team would thrive in AOL.</li>
+        <li><span class='font-bold'>Solution Seekers</span>: Members who are proactive, ready to tackle challenges head-on, and are always looking for innovative solutions.</li>
+        <li><span class='font-bold'>Community-Focused Individuals</span>: Those who believe in giving back. This includes experienced developers who've landed jobs and are keen on mentoring, guiding, and sharing their experiences with newcomers.</li>
+        <li><span class='font-bold'>Adaptable and Resilient</span>: The tech world is ever-evolving, and projects can be unpredictable. Members who are adaptable to new technologies, methodologies, and can handle setbacks with a positive attitude would be ideal.</li>
+        <li><span class='font-bold'>Ethical and Respectful</span>: Given the diverse nature of most tech communities. it's crucial for members to uphold a standard of respect, understanding, and integrity, ensuring a safe and inclusive environment for all.</li>
+      </ol>
+      <p>In essence, AOL would be perfect for those who are keen on growing both technically and personally, while valuing the strength of a supportive community.</p>
+      <h3 class='text-xl mt-4'>What happens if you don't get accepted?</h3>
+      <p>If you are not accepted, we will try to give you feedback over email. Our goal is your success. It's common for people to re-apply later and be accepted.</p>
+      "
```

Comment:
Feels maybe kinda iffy to have styles happening here? I think I'd prefer this to be "dumb" text content. Maybe there's a different way to approach it.

Reply:
Maybe we can just have a bunch of partials named after each status that holds all of the markup. And then the view just renders the partial for the current status.

Seems like this would be easier to maintain re: markup and styles.

Reply:
this is the idea I initially had, partials with all the text needed. these files should strictly be used for text, not html or markup.